# HG changeset patch
# User Rik <rik@octave.org>
# Date 1425408036 28800
#      Tue Mar 03 10:40:36 2015 -0800
# Node ID 0e0a3bbe142fd63fd852bb3a4dbe8f2cac949f37
# Parent  90299974feda87dabb86499a0bab4a8c45ebee51
Canvas.cc: Replace tabs with spaces.

* Canvas.cc: Replace tabs with spaces.

diff --git a/libgui/graphics/Canvas.cc b/libgui/graphics/Canvas.cc
--- a/libgui/graphics/Canvas.cc
+++ b/libgui/graphics/Canvas.cc
@@ -402,18 +402,18 @@ void Canvas::canvasMouseDoubleClickEvent
   if (obj.valid_object ())
     {
       graphics_object axesObj;
 
       Matrix children = obj.get_properties ().get_children ();
       octave_idx_type num_children = children.numel ();
 
       for (int i = 0; i < num_children; i++)
-	{
-	  graphics_object childObj (gh_manager::get_object (children(i)));
+        {
+          graphics_object childObj (gh_manager::get_object (children(i)));
 
           if (childObj.isa ("axes"))
             {
               graphics_object go = selectFromAxes (childObj, event->pos ());
 
               if (go)
                 {
                   axesObj = childObj;
@@ -722,18 +722,18 @@ void Canvas::canvasWheelEvent (QWheelEve
       std::string mode;
 
       graphics_object axesObj;
 
       Matrix children = obj.get_properties ().get_children ();
       octave_idx_type num_children = children.numel ();
 
       for (int i = 0; i < num_children; i++)
-	{
-	  graphics_object childObj (gh_manager::get_object (children(i)));
+        {
+          graphics_object childObj (gh_manager::get_object (children(i)));
 
           if (childObj.isa ("axes"))
             {
               graphics_object go = selectFromAxes (childObj, event->pos ());
 
               if (go)
                 {
                   axesObj = childObj;
